lib/shared/src/value.rs

```diff
@@ -20,6 +21,34 @@ pub enum Value {
     Null,
 }
 
+macro_rules! value_impl {
```

Comment:
Ah, this got snuck in from `remap::Value`. I wanted to introduce this in a follow-up PR (the rename from `as_bytes` to `as_bytes_lossy` is related to this change).

I'm happy to pull it out, but if others are fine with it, I'll sneak it in through this PR 😄

Reply:
Never mind, reverted this change for now.